cap-02/README.md

```diff
@@ -0,0 +1,221 @@
+    }
+
+Nota:
+
+    "!!" estos sirven para forzar el resultado que sea un tipo Booleano
+    
+### Texto en los canvas
+
+En caso de que el navegador no soporte los "Canva API" se creara un elemento para los `<canvas>` el cual solo tendrá propiedades básicas.
+
+    Nos dan este ejemplo:
+    
+    function supports_canvas_text() {
+    if (!supports_canvas()) { return false; }
+    var dummy_canvas = document.createElement('canvas');
+    var context = dummy_canvas.getContext('2d');
+    return typeof context.fillText == 'function';
+    }
+    
+    Nos muestra el elemento:
+    
+    if (!supports_canvas()) que es la funcion que se hizo anteriormente con esta se corrobora que soporte los canvas 
+    Esto sirve para:
+    
+    var dummy_canvas = document.createElement('canvas');
+    var context = dummy_canvas.getContext('2d');
+    
+    Crear el elemento <canva> y obtener lo que se dibujara
+    
+    Y el: 
+    
+    return typeof context.fillText == 'function'; 
+                            ^
+                            |
+                            |
+    Y al final solo chechar si tiene la funcion señalada para ver si funciona en caso de que si , es que si funciona
+    
+### VIDEO
+HTML5 tiene un elemento que es `<video>` 
+
+Este se puede usar sin necesidad de algun `<script>` 
+
+Y si el navegador no lo soporta 
+
+De igual manera que en los anteriores se puede usar Mordernizr para corroborar que el navegador soporta el elemento.
+
+
+### Formatos de video
+Debido a que existen distintos formatos de video , el navegador debe entender 
+
+Nota:
+Los formatos de video(lenguajes) se les nombran "codec"
+
+Y cada codec cuesta dinero debido a la licencia que tienen
+
+
+Los formatos de video tienen una funcion de nombre 
+**.canPlayType()** la cual regresa un valor tipo cadena dependiendo si puede o no reproducir el video
+
+* "*probably*" Puede reproducirlo
+* "*maybe*" Esta 50/50 que pueda reproducirlo
+* ""(vacio)  No lo puede reproducir
+
+
+ *return v.canPlayType('video/ogg; codecs="theora, vorbis"');*
+ Esta funcion lo que dice es que si puede reproducir los videos theora y vorbis en un contenedor de Ogg
+ 
+ De igual manera se puede hacer todo esto con Modernizr para facilitar toda la escritura de codigo.
+ 
+### Almacenamiento Local
+Nos hablan de las "*Cookies*" que son información del usuario que se almacenan en la página web gracias a JavaScript , funciona cuando se carga la pagina 
+
+En caso de que el navegador no soporte este tipo de almacenamiento lo que se almacena será indefinido
+
+Esta funcion generara una excepción si las "*cookies*" están deshabilitadas
+
+    function supports_local_storage() {
+    try {
+       return 'localStorage' in window && window['localStorage'] !== null;
+    } catch(e){
+     return false;
+     }
+    }
+    
+De igual manera se puede usar Mordernizr
+
+
+### Trabajadores de la web
+ 
+ Estos permiten que en el fondo permita correr diferentes "*threads*" al mismo tiempo 
```

Comment:
Como en este caso _threads_ está bien utilizado aquí. Solo que formalmente, hay que usar itálicas cuando escribimos este tipo de palabras.